doc/proposals/olm-cmd-workflows.md

```diff
@@ -0,0 +1,120 @@
+---
+title: README describing new command workflows for olm enabled operators
+authors:
+  - "@varshaprasad96"
+reviewers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@estroz"
+approvers:
+  - "@jlanford"
+  - "@dmesser"
+  - "@estroz"
+creation-date: 2020-04-09
+last-updated: 2020-04-09
+status: provisional
+---
+
+# README describing new command workflows for olm enabled operators
+
+## Release Signoff Checklist
+
+- \[ \] Enhancement is `provisional`
+- \[ \] Design details are appropriately documented from clear requirements
+- \[ \] Test plan is defined
+- \[ \] Graduation criteria for dev preview, tech preview, GA
+- \[ \] User-facing documentation is created.
+
+## Summary
+
+This proposal provides documentation describing the possible workflows of new commands for generating operator manifests with user inputs and publishing operator bundle image for OLM to deploy the operator.
+
+We plan to improve the developer experience for operator authors using Operator SDK with OLM by:
+1. Providing an interactive subcommand to get inputs for generating CSV.
+2. Providing a single command for generating operator manifests and for creating operator bundle image.
+
+## Motivation
+
+Currently, the `operator-sdk generate csv` command is used for generating CSVs, which sacffolds the project and writes the csv manifest on disk. This process still requires the developers to manually edit the CSV when any of the required fields are missing. Also, with the depreciation in the use of package manifests for olm, generating CSVs and creating operator bundle image does not require two separate commands. Integrating both the functionalities and providing a single command with an interactive input for collecting the required UI metadata to generate CSV as well as for building an operator bundle image will result in a better developer experience.
```

Comment:
typo: sacffolds -> scaffolds
typo: csv manifest -> CSV manifest
did you mean deprecation? instead of "depreciation"?
typo: for olm -> for OLM